keyvault/client/src/main/java/com/azure/keyvault/models/SecretRequestAttributes.java

```diff
@@ -0,0 +1,154 @@
+package com.azure.keyvault.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.time.Instant;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+
+/**
+ * The object attributes managed by the Secret service.
+ */
+public class SecretRequestAttributes {
```

Comment:
This class should not be public, it's an internal implementation detail.

Reply:
Made package-private.